Files/MultilingualResources/Files.he-IL.xlf

```diff
@@ -52,7 +52,7 @@
         </trans-unit>
         <trans-unit id="NavigationToolbarCopyPath.Text" translate="yes" xml:space="preserve">
           <source>Copy location</source>
-          <target state="translated" state-qualifier="tm-suggestion">העתק נתיב</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">העתק מיקום</target>
```

Comment:
All translated strings should be marked as translated.

Reply:
Not sure I follow. Are you addressing this specific line or in general? 

Here I took something that was translated inconsistently and changed the translation -  how should it be tagged?

Reply:
@Don-Vito I created a PR in your repository, take a look

Reply:
@Don-Vito Thank you for opening this pull request 😂
tsvietOK is requesting that you change state of translated strings from `needs-review-translation` to `translated`.

Reply:
@XPoppyX  - thanks for the PR!
@yaichenbaum - is there a description for the state-machine somewhere? Is it always switching from `new` to `translated`? We could probably add this as a Contribution section in README.

Reply:
@Don-Vito Most people do the translations through the Multilingual App Toolkit Editor which handles the state automatically, however we can definitely add some information about doing it manually.